nemoguardrails/rails/llm/config.py

```
@@ -2395,3 +1484,7 @@ def _validate_rail_prompts(rails: list[str], prompts: list[Any], validation_rail
                 raise InvalidRailsConfigurationError(
                     f"Missing a `{expected_prompt}` prompt template, which is required for the `{validation_rail}` rail."
                 )
+
+
+validate_no_config_export_shadowing(globals().keys())
+__all__ = sorted({name for name in globals() if not name.startswith("_")} | set(config_exported_names()))  # pyright: ignore[reportUnsupportedDunderAll]
```

Ruff (0.15.21)

[error] Invalid format for `__all__`, must be `tuple` or `list`

(PLE0605)